modules/python/python-dest.c

```diff
@@ -483,3 +550,19 @@ python_dd_new(GlobalConfig *cfg)
 
   return (LogDriver *)self;
 }
+
+void
+py_destination_init(void)
+{
+#define register_worker_result(value) \
```

Comment:
Why did you use a macro?

Reply:
The injection needs both the token and the string literal generated from the token. With functions, I would need to specify both.